docs/index.md

```diff
@@ -32,7 +37,7 @@ FloydHub's Team Plan is a collaborative tool for data science teams of all sizes
 
 FloydHub hosts open source Projects and Datasets that you can discover, clone, and reproduce (or reconfigure with your own Dataset).
```

Comment:
I think we can drop the community section here since the example is very old now.